CoreAutomata/TransformationExtensions.cs

```diff
@@ -35,15 +35,24 @@ public static Location Transform(this Location Location)
         {
             var scale = ScriptToScreenScale();
 
-            var scaledPoint = scale == null
-                ? Location
-                : Location * scale.Value;
+            var scaledPoint = Location * scale ?? Location;
```

Comment:
This expression is written in a confusing way for non C# developers.

I personally would prefer `= Lociation * (scale ?? 1)`. Or better yet, why not return 1 in ScriptToScreenScale()?